packages/ocaml-variants/ocaml-variants.4.12.0+domains+effects/opam

```diff
@@ -0,0 +1,65 @@
+opam-version: "2.0"
+synopsis: "OCaml 4.12.0, with support for multicore domains and effects"
+maintainer: "platform@lists.ocaml.org"
+depends: [
+  "ocaml" {= "4.12.0" & post}
+  "base-unix" {post}
+  "base-bigarray" {post}
+  "base-threads" {post}
+  "base-domains" {post}
+  "base-effects" {post}
+  "ocaml-option-nnp"
+]
+conflict-class: "ocaml-core-compiler"
+flags: compiler
+setenv: CAML_LD_LIBRARY_PATH = "%{lib}%/stublibs"
+build: [
+  [
+    "./configure"
+    "--prefix=%{prefix}%"
+    "--with-afl" {ocaml-option-afl:installed}
+    "--disable-native-compiler" {ocaml-option-bytecode-only:installed}
+    "--disable-force-safe-string" {ocaml-option-default-unsafe-string:installed}
+    "DEFAULT_STRING=unsafe" {ocaml-option-default-unsafe-string:installed}
+    "--disable-flat-float-array" {ocaml-option-no-flat-float-array:installed}
+    "--enable-flambda" {ocaml-option-flambda:installed}
+    "--enable-frame-pointers" {ocaml-option-fp:installed}
+    "CC=cc" {!ocaml-option-32bit:installed & !ocaml-option-musl:installed & (os="openbsd"|os="macos")}
+    "CC=musl-gcc" {ocaml-option-musl:installed & os-distribution!="alpine"}
+    "CFLAGS=-Os" {ocaml-option-musl:installed}
+    #"CC=gcc -m32" {ocaml-option-32bit:installed & os="linux"}
+    #"CC=gcc -Wl,-read_only_relocs,suppress -arch i386 -m32" {ocaml-option-32bit:installed & os="macos"}
+    "ASPP=cc -c" {!ocaml-option-32bit:installed & !ocaml-option-musl:installed & (os="openbsd"|os="macos")}
+    "ASPP=musl-gcc -c" {ocaml-option-musl:installed & os-distribution!="alpine"}
+    #"ASPP=gcc -m32 -c" {ocaml-option-32bit:installed & os="linux"}
+    #"ASPP=gcc -arch i386 -m32 -c" {ocaml-option-32bit:installed & os="macos"}
+    #"AS=as --32" {ocaml-option-32bit:installed & os="linux"}
+    #"AS=as -arch i386" {ocaml-option-32bit:installed & os="macos"}
+    #"--host=i386-linux" {ocaml-option-32bit:installed & os="linux"}
+    #"--host=i386-apple-darwin13.2.0" {ocaml-option-32bit:installed & os="macos"}
+    #"PARTIALLD=ld -r -melf_i386" {ocaml-option-32bit:installed & os="linux"}
+    # 32bit options above commented out just to reduce diff with ocaml-variants.4.12.0+options
+    "LIBS=-static" {ocaml-option-static:installed}
+    "--disable-warn-error"
+  ]
+  [make "-j%{jobs}%"]
+]
+install: [make "install"]
+url {
+  src: "git+https://github.com/ocaml-multicore/ocaml-multicore.git#4.12+domains+effects"
```

Comment:
The use of a Git URL here is tripping opam-dune-lint... this rings a bell, but I can't remember the full story. @kit-ty-kate?

Reply:
thanks!